### DISCUSSION TITLE
How do I set Tabby to use powershell as default ?

### DISCUSSION THREAD

**Discussion by iFAXec** [Eugeny/tabby#8191] [Q&A] [answered]
I would like Tabby to open up PowerShell on default when I open this terminal. Is there a way to default it? I checked the profiles and connection section, but there is no way to set Tabby to open PowerShell as default, it just allows me to create a new profile using various shell options like bash, PowerShell, Ubuntu, etc. 

Your reply is appreciated.

**Comment by Eugeny** [comment]
Settings -> Profiles -> Default profile

**Reply by iFAXec** [reply]
So if I understand this correctly, I have to keep creating a new profile for every path so that I can use it after opening Tabby. There is no way t directly open Tabby with PowerShell or Bash in one go. 

That's because when I open Tabby, it opens the first screenshot.
The second screenshot is just the way to create multiple profiles under settings.

![2Screenshot 2023-04-07 183035](https://user-images.githubusercontent.com/87900414/230651827-4613f1ad-f034-4e71-9730-6bfcec51113a.png)


![Screenshot 2023-04-07 182437](https://user-images.githubusercontent.com/87900414/230651036-f0fe44c9-32e2-41eb-85f9-aaec40b731d3.png)

**Reply by Eugeny** [reply, marked as answer]
There's supposed to be a dropdown list where you can select the default profile - could you please check whether there are any errors showing up in the debug log? https://github.com/Eugeny/terminus/wiki/Debug-logs

<img width="612" alt="image" src="https://user-images.githubusercontent.com/161476/230654444-aaa92723-6f74-461e-b97d-902e8d20b4d6.png">

**Reply by iFAXec** [reply]
Got it thanks. 

The black theme that I was using didn't highlight the default profile box and it completely overshadowed it. I had to change my theme to see the box and access dropdown menus. 

Managed to sort this out. Appreciate your help.

**Reply by lawweiliang** [reply]
> Got it thanks.
> 
> The black theme that I was using didn't highlight the default profile box and it completely overshadowed it. I had to change my theme to see the box and access dropdown menus.
> 
> Managed to sort this out. Appreciate your help.

I have same issue previously as well. Can't see the default profile drop-down list. 
-  https://github.com/Eugeny/tabby/discussions/8151

@iFAXec 
Do remember to mark eugene comment as `answered` and `close` this discussion yeah.
Thanks
